pkg/image/export/main.go

```diff
@@ -91,12 +91,12 @@ func run(systemChartPath, chartPath string, imagesFromArgs []string) error {
 
 	k3sUpgradeImages := getK3sUpgradeImages(rancherVersion, data.K3S)
 
-	targetImages, targetImagesAndSources, err := img.GetImages(systemChartPath, chartPath, k3sUpgradeImages, imagesFromArgs, linuxInfo.RKESystemImages, img.Linux)
+	targetImages, targetImagesAndSources, err := img.GetImages(systemChartPath, chartPath, rancherVersion, k3sUpgradeImages, imagesFromArgs, linuxInfo.RKESystemImages, img.Linux)
```

Comment:
@rmweir can rancherVersion still be `DEV` ?

Reply:
@cbron if it is "dev" it will be converted to 2.x.99 by kd.GetRancherVersion().